MBBSEmu.Tests/ExportedModules/Majorbbs/cncchr_Tests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using MBBSEmu.Memory;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class cncchr_Tests : MajorbbsTestBase
+    {
+        private const int CNCCHR_ORDINAL = 122;
+
+        [Theory]
+
+        //Simple commands with nxtcmd == input
+        [InlineData("123", 1, '2', 2)]
+        [InlineData("1 2", 0, '1', 2)]
+        [InlineData("2 1", 2, '1', 3)]
+        [InlineData("123 456", 4, '4', 5)]
+        [InlineData("\0", 0, '\0', 0)]
+        public void cncchr_Test(string inputString, ushort nxtcmdStartingOffset, char expectedResult, ushort expectedNxtcmdOffset)
```

Comment:
follow-up: it'd be really killer to have some sort of more comprehensive test that includes cncchr + cncyesno in a combination